packages/typescript/src/symbols/ts-output-symbol.ts

```diff
@@ -78,3 +79,7 @@ export function createTSSymbol(options: createTsSymbolOptions): TSOutputSymbol {
 
   return sym;
 }
+
+export function isNullish(sym: OutputSymbol): boolean {
```

Comment:
I generally don't find this kind of helper useful, the bitwise op is pretty terse. No real problem with it if we can avoid making it public API though.

Reply:
Yeah tat's fair